### PR TITLE
feat: Update dependencies

### PR DESCRIPTION
**Pull request by shiipou**
feat: Upgrade lenra_ui_runner to v1.0.0-beta.91 feat: receive response from listeners (#191)

### CODE REVIEW

**Review by jonas-martinez — DISMISSED** [review]

File: client/pubspec.lock [hunk 1]
```diff
@@ -35,7 +35,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.9.0"
```
Comment (taorepoara) [inline client/pubspec.lock:38]: It is downgrading the lock file...

**Review by taorepoara — CHANGES_REQUESTED** [review]

File: client/pubspec.lock (hunk 1 above)
Comment (jonas-martinez) [reply client/pubspec.lock]: @taorepoara Fixed

**Comment by shiipou** [comment]
update lock

**Comment by shiipou** [comment]
feat: Upgrade lenra_ui_runner to v1.0.0-beta.93

**Comment by shiipou** [comment]
feat: Upgrade lenra_ui_runner to v1.0.0-beta.94

**Comment by shiipou** [comment]
feat: Upgrade lenra_ui_runner to v1.0.0-beta.95

**Comment by shiipou** [comment]
feat: Upgrade lenra_ui_runner to v1.0.0-beta.96

**Comment by shiipou** [comment]
feat: Upgrade lenra_ui_runner to v1.0.0-beta.97

**Pull request closed without merging**